index.js

```diff
@@ -43,22 +52,37 @@ function start () {
     }
 
     argv.inputPath = path.isAbsolute(argv.input) ? argv.input : path.join(process.cwd(), argv.input)
-    const results = require(argv.inputPath)
-    const report = buildReport(results)
-    writeReport(report, argv.outputPath, (err) => {
-      if (err) console.err('Error writting report: ', err)
-      else console.log('Report written to: ', argv.outputPath)
-    })
-  } else {
-    const concatStream = concat((res) => {
-      const results = JSON.parse(res.toString())
-      const report = buildReport(results)
-      writeReport(report, argv.outputPath, (err) => {
-        if (err) console.err('Error writting report: ', err)
-        else console.log('Report written to: ', argv.outputPath)
+    if (argv.compare) {
```

Comment:
if argv.compare doesn't exist, this will cause issues. just remove this if and do argv.compare = argv.compare || []